part1/local-kafka.md

```diff
@@ -12,6 +12,12 @@ We can start ZooKeeper with the default configuration file, by running this comm
 ./bin/zookeeper-server-start.sh ./config/zookeeper.properties
 ```
 
+**NOTE:** If you're working on Windows with `bash`, before executing the above command you should edit `config/zookeeper.properties` in order to indicate the absolute path in Windows format for the *dataDir* — usually is used the default temp folder. For example:
```

Comment:
`usually is used the default temp folder` doesn't sound quite right. Maybe we can drop that part of the sentence?

Reply:
I think so.. item corrected!